### PR TITLE
[CAD-4086] Hookup to cyrus ci/cd

### PR DESCRIPTION
**Pull request by gaigepr**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,61 @@
+#!groovy
+
+def GIT_BRANCH = ''
+def IMAGE_REF = ''
+def IMAGE_TAG = ''
+def VERSION = ''
+def NAMESPACE = ''
```
Comment (squidlegs) [inline Jenkinsfile]: not needed

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,61 @@
+#!groovy
```
Comment (squidlegs) [inline Jenkinsfile:1]: gaigepr wants to merge 5 commits into master from add-jenkinsfile

Comment (squidlegs) [reply Jenkinsfile:1]: we should at least do RC and master, if not release

File: Jenkinsfile (hunk 1 above)
Comment (gaigepr) [reply Jenkinsfile]: addressed

File: Jenkinsfile (hunk 2 above)
Comment (gaigepr) [reply Jenkinsfile:1]: done

**Review by squidlegs — APPROVED** [review]
approved when passing

**Comment by gomoto** [comment]
What about a staged Dockerfile?

**Pull request merged**